src/validators/block_id/mod.rs

```diff
@@ -103,26 +103,48 @@ pub fn display_current_validators_info(
     );
 
     let mut table = Table::new();
-    table.set_titles(prettytable::row![Fg=>"Validator Id", "Stake", "Online", "Blocks produced", "Blocks expected", "Chunks produced", "Chunks expected"]);
+    table.set_titles(prettytable::row![Fg=>"#", "Validator Id", "Stake", "Online", "Blocks produced", "Blocks expected", "Chunks produced", "Chunks expected", "Endorsements produced", "Endorsements expected"]);
 
-    for validator in &current_validators {
+    for (index, validator) in current_validators.into_iter().enumerate() {
         let online = if validator.num_expected_blocks + validator.num_expected_chunks == 0 {
```

Comment:
Nit: when I see this, I always go and check if the numbers may be negative
A better style would be to explicitly check what we need (if a == 0 and b == 0)

Reply:
Done